tema21ian/src/ExerciseTwo/DegreesTest.java

```diff
@@ -0,0 +1,46 @@
+package ExerciseTwo;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+ class DegreesTest {
+        Degrees degrees;
+
+    {
+        new Degrees();
```

Comment:
this line doesn't do anything helpful. You create a new instance of Degrees, but you don't store that new instance anywhere.
Also all your methods in Degrees are static ones, so you don't need an instance to access those.